### PR TITLE
DRAFT 'What is a Node' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
This is very much a draft. A few things:

1. I'm not sure of the links 
2. Should we include what a full node is and why it’s beneficial? 
3. Although implicit, I think we can emphasise a bit more why running a node is important. Also, the second part of this article ('Do I need to run one?'), hasn't been explicitly answered.

### CODE REVIEW

**Comment by AbcSxyZ** [comment]
Everything is based on top of https://github.com/dogecoin/dogecoin, called Dogecoin Core, the "specialized software". At each new version they are doing a [release](https://github.com/dogecoin/dogecoin/releases) where you can find executables for your specific computer. dogecoin.com executables of Dogecoin Core are just links to GitHub ;)

Agree it can be interesting to speak about difference between full node (Dogecoin Core) & relation with spv wallet (Multidoge). Could be great to say that miners are specifics type of node.

**Comment by MishaBoar** [comment]
Completing this - sending updated version soon.

**Comment by RichieJohn** [comment]
> Completing this - sending updated version soon.

Great! :)

**Comment by MishaBoar** [comment]
I have provided a revised and substantially expanded version of the article! Please review it, especially for clarity/typos. Thanks!

File: content/dogepedia/articles/what-is-a-node.md [hunk 1]
```diff
@@ -5,4 +5,43 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-A fairly accessible introduction to what a node is, and the role of nodes in the network.
+The Dogecoin network is a peer-to-peer payment network consisting of thousands of computers called nodes. Each of these nodes is running a specialized software, called Dogecoin Core. Transaction information is recorded into a database, split into blocks chained to each other, and distributed and replicated across the nodes of the network: this database is known as the *Dogecoin blockchain*.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-node.md:8]: delete "a" before "specialized" ?

File: content/dogepedia/articles/what-is-a-node.md [hunk 2]
```diff
@@ -5,4 +5,43 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-A fairly accessible introduction to what a node is, and the role of nodes in the network.
+The Dogecoin network is a peer-to-peer payment network consisting of thousands of computers called nodes. Each of these nodes is running a specialized software, called Dogecoin Core. Transaction information is recorded into a database, split into blocks chained to each other, and distributed and replicated across the nodes of the network: this database is known as the *Dogecoin blockchain*. 
+
+In practice, this results in transaction information being shared across the entire network and, subsequently, not needing to be stored in one centralized place. This is why cryptocurrencies like Dogecoin are often referred to as being *decentralized*.
+
+In short, a node is a software that is running on a computer that is connected to the Dogecoin network. This software communicates with other nodes on the network and in doing so keeps the distributed ledger in sync and up-to-date, in addition to providing new nodes that need to download the full Dogecoin blockchain with information about past blocks.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-node.md:12]: delete "a" before "software" ?

**Review by RichieJohn — COMMENTED** [review]
@MishaBoar This is great! Only a few small grammatical suggestions that I've made in-text. Perhaps these suggestions are redundant, though? That is, although I haven't seen the word "a" placed before the word "software" before, the more I think about it the more it seems reasonable! :)

**Review by MishaBoar — APPROVED** [review]

**Pull request merged**